src/theme/styled.d.ts

```diff
@@ -3,7 +3,7 @@ import { getTheme } from './index'
 type InferredTheme = ReturnType<typeof getTheme>
 
 declare module 'styled-components/macro' {
-  export interface DefaultTheme extends InferredTheme {
+  interface DefaultTheme extends InferredTheme {
```

Comment:
While this makes sense, I wonder why official documentation (and every single example out there) go with "export" https://styled-components.com/docs/api#create-a-declarations-file

One thing that comes to my mind is being able to actually import a "DefaultTheme" type from "styled-components/macro" directly (not from this file), which could confuse the linter. 

Can you check `DefaultTheme` on this line:
https://github.com/Uniswap/interface/blob/91157b7a43f0c7c75238ac716298bbc72b30810a/src/components/Badge/index.tsx#L3
has `any` or an actual type? I am afraid it may be `any`.

Reply:
if it's in the docs i can just ignore this line